gamestonk_terminal/stocks/discovery/disc_controller.py

```diff
@@ -352,6 +356,90 @@ def call_under(self, other_args: List[str]):
         except Exception as e:
             print(e, "\n")
 
+    def call_gtech(self, other_args: List[str]):
+        """Process gtech command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="gtech",
+            description="Print up to 25 top ticker losers. [Source: Yahoo Finance]",
```

Comment:
You forgot to update this description. I would pick the description from here https://finance.yahoo.com/screener